### ISSUE TITLE
Works great with <script> tags but failes with npm package

### ISSUE THREAD

**Issue by enahum** [twilio/twilio-conversations.js#9]
I've been playing around with the Twilio Video and if I use the script tags I can create a `new Twilio.AccessManager`, `new Twilio.Conversations.Client` and so on.

If I try to load the library through npm and webpack, browserify, whatever the same code just doesn't work.

Any pointers on this?

**Comment by enahum** [comment]
I made it work but I had to comment this line https://github.com/twilio/twilio-conversations.js/blob/01bf25157561d64413336b5e776d5af1165e8f6e/lib/client.js#L5 because for some reason `require('twilio-common')` returns `undefined` while loading with webpack.

Feedback on this?

**Comment by markandrus** [comment]
Hi @enahum, and thanks for writing in.

I think what you are seeing is due to the browser override we have in place for twilio-common. It's defined [here in package.json](../blob/01bf25157561d64413336b5e776d5af1165e8f6e/package.json#L66) and points to [src/twilio-common.js](../blob/01bf25157561d64413336b5e776d5af1165e8f6e/src/twilio-common.js). So the [line you point out](../blob/01bf25157561d64413336b5e776d5af1165e8f6e/lib/client.js#L5) ends up trying to require a browser global (`Twilio`) that does not exist when you go the Webpack/Browserify route.

We created the browser override for &lt;script&gt; tag users, thinking it would be nice if they could include twilio-common.js and twilio-conversations.js separately. But since then we've decided against it (makes updating harder, doesn't really give the user anything, etc.).

So I think in the next release we will change this: the &lt;script&gt; version of twilio-conversations.js will begin bundling twilio-common.js, and using Webpack/Browserify should "just work".

**Comment by enahum** [comment]
Thanks @markandrus so far I forked the project and made the change to that line of code and now it works with webpack.

BTW you should update dependancies and review tests cause two of them were failing for me

**Comment by markandrus** [comment]
@enahum this should be fixed in 0.13.6. Can you retest? Thanks! (also, integration tests should be a little more robust now)

**Comment by newtriks** [comment]
I'm having troubles with this project and webpack. It's the only dependency in this project that seems to be a problem thus far and is based of my trying various versions of node.

I've installed using: `"twilio-video": "git+ssh://git@github.com/twilio/twilio-video.js.git#master"` and looking at the package.json version I see `"version": "1.0.0-dev"`. When bundling my app with Webpack (V1) for some reason I get the following errors:

```
ERROR in ./~/twilio-video/package.json
Module parse failed: .../node_modules/twilio-video/package.json Unexpected token (2:8)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (2:8)
    at Parser.pp$4.raise (.../node_modules/acorn/dist/acorn.js:2221:15)
 @ ./~/twilio-video/lib/index.js 3:14-35

ERROR in ./~/sip.js/package.json
Module parse failed: .../node_modules/sip.js/package.json Unexpected token (2:8)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (2:8)
    at Parser.pp$4.raise (.../node_modules/acorn/dist/acorn.js:2221:15)
    at Parser.pp.unexpected
 @ ./~/sip.js/src/SIP.js 9:10-36
```
